leptos_dom/src/components/errors.rs

```diff
@@ -20,7 +20,7 @@ where
           Some(errors) => {
             let id = HydrationCtx::id();
             errors.update({
-              let id = id.clone();
+              let id = id;
```

Comment:
This is now redundant, which makes the added `{ ... }` within `.update()` redundant.